lib/importer.py

```diff
@@ -633,11 +633,16 @@ def execImport(handle, options):
             if xbmcmediaimport.shouldCancel(handle, itemsProgress, itemsProgressTotal):
                 return
 
-            item = Api.toFileItem(plexServer, plexItem, mediaType, plexLibType)
-            if not item:
-                continue
+            try:
+                item = Api.toFileItem(plexServer, plexItem, mediaType, plexLibType)
+                if not item:
+                    continue
 
-            items.append(item)
+                items.append(item)
+
+            except plexapi.exceptions.BadRequest as err:
```

Comment:
Shouldn't we move this check to the part in `Api.toFileItem()` where it is actually happening to narrow down the `try-except` area?

Reply:
As discussed issue happens when accessing specific properties later in `convertDateTimeToDbDateTime`. I guess it is better to keep it as is since the same error can occur with any of the properties the item holds. No exception is raised in `toFileItem` itself and the implementation:

https://github.com/Montellese/mediaimporter.plex/blob/master/plex/api.py#L266

shows `fillVideoInfos` doesn't have any error handling